### PR TITLE
Bump up GoogleDataTransport version to 8.1.0

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**
Bump up version for #7064.
Reference to FirebasePerformance's changes https://github.com/FirebasePrivate/firebase-ios-sdk/pull/734#pullrequestreview-542951310

### CODE REVIEW

File: GoogleDataTransport.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'GoogleDataTransport'
-  s.version          = '8.0.1'
+  s.version          = '8.2.0'
```
Comment (paulb777) [inline GoogleDataTransport.podspec:3]: Any reason for skipping 8.1.0? I believe it should be 8.1.0

Comment (maksymmalyhin) [reply GoogleDataTransport.podspec:3]: Yeah, I think it should be 8.1.0 (being in sync with the changelog as well).

Comment (jeremyjiang-dev) [reply GoogleDataTransport.podspec]: Sorry. I meant 8.1.0. Fixed.

**Review by maksymmalyhin — APPROVED** [review]
LGTM, thank you for the PR!

**Review by paulb777 — APPROVED** [review]

**Comment by jeremyjiang-dev** [comment]
> LGTM, thank you for the PR!

Thanks for reviewing it! Merging the PR now.

**Pull request merged**